app/Settings.php

```diff
@@ -19,4 +19,9 @@ public function routeNotificationForSlack($full = false)
 
         return null;
     }
+    // Return default team id for incoming tickets
+    public static function defaultTeam()
+    {   
+        return Settings::first()->default_team_id;
```

Comment:
Since this function is returning an id I would change its name to `defaultTeamId()`